cibuild.sh

```diff
@@ -13,10 +13,19 @@ if [ $? -ne 0 ]; then
     exit 1
 fi
 
+# NuGet on mono crashes about every 5th time we run it.  This is causing
+# Linux runs to fail frequently enough that we need to employ a 
+# temporary work around.  
 echo Restoring NuGet packages
-mono src/.nuget/NuGet.exe restore src/Roslyn.sln
+i=5
```

Comment:
This makes probability of a failure 3.2 * 10^-4. If we increase the number of attempts to 10, it becomes 10^-5.